docs/cli/README.md

```diff
@@ -150,6 +150,12 @@ Using the command `neutrino start` builds a project in development mode, also st
 watcher depending on the middleware or configuration used. This command sets the `NODE_ENV` environment variable to
 `development` by default.
 
+This command also receives a flag `--dashboard` to add the [webpack-dashboard](https://github.com/FormidableLabs/webpack-dashboard) plugin. To use the plugin, the start script in package.json can be modified:
+
+```json
+"start": "webpack-dashboard -- neutrino start --dashboard"
```

Comment:
Is the `webpack-dashboard` command really needed here? Is there a way we can make this work with **only** `neutrino start --dashboard`?

Reply:
I couldn't find a way to make it work that way.

> In the latest version, you can either run your app, and run webpack-dashboard independently (by installing with npm install webpack-dashboard -g) or run webpack-dashboard from your package.json`

Reply:
Apologies, this issue used to be simpler. The only thing I can think of then to make this a reality would be to `fork` the process by calling `webpack-dashboard` with the starting script. If you're up for it, we can try experimenting with it, but it will be much more complex.

Reply:
I'm working on it. I've spawned a new child process, but I need to silence the main process' stdout. I mean, messages like these:

>✔ Development server running on: http://localhost:5000
>✔ Build completed

Is there a way to silence the output and let webpack-dashboard take over stdout?

Reply:
The CLI supports a `quiet` flag. Maybe you can set `args.quiet` to `true` when `args.dashboard` is `true`?

Reply:
Done